platform/android/java/src/org/godotengine/godot/Godot.java

```diff
@@ -937,6 +945,18 @@ public void setPaymentsManager(PaymentsManager mPaymentsManager) {
 	*/
 
 	// Audio
```

Comment:
By chance this comment is correct now, but I guess in the future we might want to add support for more permissions in the same method, so maybe it should be removed/updated?

Reply:
I'll remove it